### PR TITLE
fix(recentplaylist): fix wrong distance & add a div prevent from turning page too fast

### PR DESCRIPTION
**Pull request by YuRiJinX**
## Recent Play List
- BUG: wrong distance on window width > 1355
- add a div prevent from closing the play list by turning pages too fast

### CODE REVIEW

File: src/renderer/components/PlayingView/RecentPlaylist.vue [hunk 1]
```diff
@@ -330,6 +341,8 @@ export default {
         position: relative;
         background-color: rgba(255,255,255,0.1);
       }
+      .next-page {
+      }
```
Comment (ipy) [inline src/renderer/components/PlayingView/RecentPlaylist.vue]: ？

**Review by ipy — CHANGES_REQUESTED** [review]

File: src/renderer/components/PlayingView/RecentPlaylist.vue [hunk 2]
```diff
@@ -56,6 +56,14 @@
         @mouseupItem="itemMouseup"
         @mouseoutItem="itemMouseout"
         @mouseoverItem="itemMouseover"/>
+      <div class="next-page"
+        v-if="thumbnailNumber < numberOfPlaylistItem"
+        @mouseup.stop=""
```
Comment (ipy) [inline src/renderer/components/PlayingView/RecentPlaylist.vue:61]: ？

Comment (YuRiJinX) [reply src/renderer/components/PlayingView/RecentPlaylist.vue:61]: 说因为下一页点太快点到最后一页没东西的时候播放列表直接就关掉了。。然后要加个东西挡着

Comment (YuRiJinX) [reply src/renderer/components/PlayingView/RecentPlaylist.vue:61]: @ipy

**Review by ipy — APPROVED** [review]

**Pull request merged**